framework/include/executioners/PicardSolve.h

```diff
@@ -76,6 +76,11 @@ class PicardSolve : public SolveObject
     _picard_self_relaxed_variables = vars;
   }
 
+  /**
+   * Whether sub-applications are automatically advanced no matter what happens during their solves
+   */
+  bool autoAdvance() const;
+
```

Comment:
Sounds we should use `forceAutoAdvance`?

Reply:
`forceAutoAdvance` sounds like it would be a setter-type method to me. This is just querying the sate of whether we are auto-advancing